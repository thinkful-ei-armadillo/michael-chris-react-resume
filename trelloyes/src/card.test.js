import React from 'react'; 
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './card.js'; 

describe('Card component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card key={1} title={'first card'} content={'lorem ipsum'}/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
}); 

it('renders the UI as expected', () => {
    const tree = renderer
    .create(<Card key={1} title={'First List'} content={'test'}/>)
    .toJSON();
    expect(tree).toMatchSnapshot();  
});