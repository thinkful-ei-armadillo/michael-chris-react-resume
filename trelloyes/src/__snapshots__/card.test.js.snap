// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    First List
  </h3>
  <p>
    test
  </p>
</div>
`;
